config.global.yaml

```diff
@@ -2,7 +2,10 @@ apiVersion: v1
 data:
   body-size: "0"
   # Change redirect code to 301 as 308 is not understood by social media bots
-  http-redirect-code: 301
+  http-redirect-code: "301"
```

Comment:
this will not work if we apply this on production.

Reply:
mistake